backend/apps/account/admin.py

```diff
@@ -117,12 +97,39 @@ class CustomUserAdmin(UserAdmin):
             },
         ),
     )
+    readonly_fields = ("date_joined", "last_login")
+    inlines = (StudentInline,)
+
     list_filter = (
         "is_staff",
         "is_superuser",
         "is_active",
         "groups",
+        "invitation",
         UppercaseEmailFilter,
         ECNantesDomainFilter,
         NoPasswordFilter,
     )
+
+    @admin.action(description="Send reminder to upgrade account.")
+    def send_reminder(self, request, queryset: list[User]):
+        current_site = get_current_site(request)
+        upgrade_link = f"https://{current_site.domain}" + reverse(
+            "account:upgrade-permanent"
+        )
```

Comment:
```suggestion
        upgrade_link = request.build_absolute_uri(reverse(
            "account:upgrade-permanent"
        ))
```

Utiliser la méthode `build_request_uri` pour utiliser http ou https suivant qu'on est en dev ou en prod